Assets/Scripts/Managers/LevelController.cs

```diff
@@ -2,44 +2,18 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class LevelController : MonoBehaviour
-{
-    public static LevelController lController;
+/// 
+/// This script must be deleted in the near future!
+/// 
 
-    [SerializeField]
-    GameObject SpawnPoint;
-    Vector3 spawnPoint;
 
-    //public transform playerSpawn;
 
-    void Awake()
-    {
-        if (lController != null)
-        {
-            Destroy(lController.gameObject);
-            lController = this;
-        }
-        else
-        {
-            lController = this;
-        }
-    }
+public class LevelController : MonoBehaviour
+{
+    public static LevelController lController;
 
     void Start()
     {
-        spawnPoint = SpawnPoint.transform.position;
-        if(!SpawnPoint)
-        {
-            Debug.Log("Failed to find the spawnpoint");
-        }
-        if(Player.player != null)
-        {
-            Player.player.transform.position = spawnPoint;
-        }
-        else
-        {
-            Debug.Log("Failed to find the player");
-        }
-
+        Debug.Log("ATTENTION! Level controller is present. Shouldn't be used anymore!");
```

Comment:
I would use debug.assert or at least .logError for something important like this.